src/models/ActivitySchema.ts

```diff
@@ -1,8 +1,8 @@
 import * as Joi from 'joi';
```

Comment:
Please see my comment in the Activity.ts regarding definition of lists locally, I think it would be cleaner to refer directly in the schema itself...

```
  endTime: Joi.string().optional().allow(null),
  waitReason: Joi.array().items([
    WaitReason.WAITING_FOR_VEHICLE,
    WaitReason.BREAK,
    WaitReason.ADMIN,
    WaitReason.SITE_ISSUE,
    WaitReason.OTHER
  ]).optional(),
  notes: Joi.string().allow(null)
```